Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -4689,16 +4696,22 @@ def panorama_query_logs(log_type: str, number_of_logs: str, query: str, address_
         'GET',
         params=params,
     )
+    parsed_namespace = parse_response(result)
```

Comment:
what do you thing about creating adding an argument to the `http_request` lets say: `parse_as_class`, give it a default false and if its `True` you can return the `SimpleNameSpace` or the `ResponseClass`

Reply:
That could work. Let me look into it.